client/executor/src/native_executor.rs

```diff
@@ -170,7 +176,7 @@ where
 	/// runtime is invalidated on any `panic!` to prevent a poisoned state. `ext` is already
 	/// implicitly handled as unwind safe, as we store it in a global variable while executing the
 	/// native runtime.
-	fn with_instance<R, F>(
```

Comment:
You've added `allow_missing_host_functions` to `self` while this function (and other ones) still take `allow_missing_host_functions` as an argument.

Reply:
Yeah, I fix it.

Reply:
This is still not fixed?

Reply:
Sorry, fixed now.